lib/posts.js

```diff
@@ -73,6 +73,16 @@ export async function getPostData(id) {
   }
 }
 
+export async function getAllCompanyMembers() {
+  const fileNames = fs.readdirSync(PEOPLE_DIRECTORY);
+  const namesOnly = fileNames.map(fileName => fileName.replace(/\.mdx$/, ''));
+
+  let people = [];
```

Comment:
```suggestion
  const people = [];
```

Since you dont modify the variable anymore you can use const in this case.